.github/workflows/lagom-template.yml

```diff
@@ -121,7 +120,7 @@ jobs:
             -DCMAKE_CXX_COMPILER=${{ steps.build-parameters.outputs.host_cxx }}
 
       - name: Create Build Environment
-        if: ${{ inputs.fuzzer == 'FUZZ' }}
+        if: ${{ inputs.build_preset == 'Fuzzer_CI' }}
```

Comment:
`inputs.build_preset` is `Fuzzer_CI` here, but the preset is named `Fuzzers_CI`. Can we keep this consistent? Also allows for direct usage of the `inputs.build_preset` variable in the cmake invocation.

Reply:
Whoops, that's a pure typo that causes errors for sure. If inputs.build_preset isn't the same as what I'm checking against, that's a bug.